src/AppBundle/Resources/views/themes/app/search/search.html.twig

```diff
@@ -16,6 +16,15 @@
                         {% if pager.nbResults == 0 %}
                             <div class="result-message result-message-error">
                                 <h2>{{ 'ngsite.search.no_results'|trans({'%searchText%': search_text}) }}</h2>
+
+                                {% if search_suggestion is not empty %}
+                                    <p>
+                                        {{ 'ngsite.search.did_you_mean'|trans }}
+                                        "<a href="{{ path('ngsite_content_search', {'searchText': search_suggestion}) }}">
```

Comment:
Quotes should go inside `<a></a>`, no?

Reply:
@emodric TBH, I don't know what I want, not an UX expert :) this whole output is just for example, it can and should be probably tweaked later (eg. the link is not visible because it has the same styling as the text).